### PR TITLE
Feature: APP-1324 - Decode update settings & DAO metadata actions

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- adds decoders for update settings & metadata actions
- adds Accordion cards for both actions (old and new toggle out of scope)
- fixes Accordion card invalid html structure
- removes unnecessary `useDaoToken` call in Proposal Details page

Note: This is waiting for new version of SDK which will provide the `votingMode` property in the decoded vote settings & fix failure to decode DAO metadata.

Task: [APP-1324](https://aragonassociation.atlassian.net/browse/APP-1324)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1324]: https://aragonassociation.atlassian.net/browse/APP-1324?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jamesej — CHANGES_REQUESTED** [review]
Hi Fabrice, tried a couple of settings changes. The first time I changed the description and added a minute to the min duration. The flow was fine but the settings block was empty on Review Proposal. The transaction failed for this one. I tried again with just the description change. Again the settings block was empty, but the transaction to publish succeeded.

**Comment by Fabricevladimir** [comment]
@jamesej , the task is for decoding the proposal actions from the SDK, so the actions will only be seen on the Proposal Details page, not Proposal Review. That's why I added the dummy settings for the review. I can update this PR with it on Monday or alternatively create a new task to handle it considering how big this PR is already. Thoughts?

**Review by jamesej — APPROVED** [review]
OK all good then thanks

File: packages/web-app/src/utils/types.ts [hunk 1]
```diff
@@ -245,14 +249,29 @@ export type ActionMintToken = {
   };
 };
 
+export type ActionUpdatePluginSettings = {
+  name: 'modify_settings';
```
Comment (WissenIstNacht) [inline packages/web-app/src/utils/types.ts]: Is settings not too generic? Will we not get a different type of settings for the multisig, for example?

**Review by WissenIstNacht — APPROVED** [review]
Thanks!

There's no technical issues, so I'll approve and leave it up to judge whether changing the name is necessary.

**Pull request merged**